desktop/src/Desktop/Frontend.hs

```diff
@@ -184,7 +184,9 @@ lockScreen xprv = setupDiv "fullscreen" $ divClass "wrapper" $ setupDiv "splash"
 
     e <- confirmButton (def & uiButtonCfg_type ?~ "submit") "Unlock"
     (_help, restore) <- setupDiv "button-horizontal-group" $ do
-      help <- uiButton def $ text "Help" -- TODO where does this go?
+      help <- uiButton def $ do
+        elAttr "img" ("src" =: static @"img/launch_dark.svg" <> "class" =: "button__text-icon") blank
+        text "Help" -- TODO where does this go?
```

Comment:
It's worth noting that the help button is still a placebo button. :slightly_smiling_face: